CVE-2021-22205/cvex.yml

```diff
@@ -0,0 +1,6 @@
+blueprint: ubuntu2204
+ubuntu:
+  playbook: ubuntu.yml
+  command:
+    - "sudo docker ps -a"
+    - "python /opt/CVE-2021-22205/poc.py http://localhost:8080 \"touch /tmp/success\""
```

Comment:
In the new version of cvex.yml based off ubuntu2204-ubuntu2204 please also run ls /tmp/success on ubuntu1. cvex.yml will look like this:
```
ubuntu2:
...
  command: "python /opt/CVE-2021-22205/poc.py http://localhost:8080 \"touch /tmp/success\""
...
ubuntu1:
...
  command: ls /tmp/success
```

This way we can verify that the file has been successfully created.